cmd/cri-containerd/options/options.go

```diff
@@ -88,19 +84,45 @@ type Config struct {
 	StatsCollectPeriod int `toml:"stats_collect_period" json:"statsCollectPeriod,omitempty"`
 	// SystemdCgroup enables systemd cgroup support.
 	SystemdCgroup bool `toml:"systemd_cgroup" json:"systemdCgroup,omitempty"`
+	// SkipImageFSUUID skips retrieving imagefs uuid.
+	// TODO(random-liu): Remove this after we find a generic way to get imagefs uuid.
+	SkipImageFSUUID bool `toml:"skip_imagefs_uuid" json:"skipImageFSUUID,omitempty"`
+}
+
+// CRIConfig contains toml config related to CRI service.
+// TODO(random-liu): Make this an internal config object when we no longer support cri-containerd
+// standalone mode. At that time, we can clean this up.
+type CRIConfig struct {
+	// PluginConfig is the config for CRI plugin.
+	PluginConfig
+	// ContainerdRootDir is the root directory path for containerd.
+	ContainerdRootDir string `toml:"-" json:"containerdRootDir,omitempty"`
+	// ContainerdEndpoint is the containerd endpoint path.
+	ContainerdEndpoint string `toml:"-" json:"containerdEndpoint,omitempty"`
+	// SocketPath is the path to the socket which cri-containerd serves on.
+	// TODO(random-liu): Remove SocketPath when no longer support cri-containerd
+	// standalone mode.
+	SocketPath string `toml:"socket_path" json:"socketPath,omitempty"`
+	// RootDir is the root directory path for managing cri-containerd files
+	// (metadata checkpoint etc.)
+	RootDir string `toml:"root_dir" json:"rootDir,omitempty"`
+}
+
+// Config contains toml config related cri-containerd daemon.
+type Config struct {
+	CRIConfig `toml:"-"`
+	// CgroupPath is the path for the cgroup that cri-containerd is placed in.
+	CgroupPath string `toml:"cgroup_path"`
 	// OOMScore adjust the cri-containerd's oom score
-	OOMScore int `toml:"oom_score" json:"oomScore,omitempty"`
+	OOMScore int `toml:"oom_score"`
```

Comment:
why remove the json definitions? I suppose this struct is going away anyhow...

Reply:
Initially we have json defination, because we print the config in `Status` info. Now these options are not part of `Status` info. So don't need to have json definition. :)